orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/token.xml.j2

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<topology>
+   <uri>https://{{ salt['pillar.get']('gateway:address') }}:8443/{{ salt['pillar.get']('gateway:path') }}/token</uri>
+   <name>cdp-token</name>
+   <gateway>
+      <provider>
```

Comment:
I wonder if you need the `webappsec` / `cors.enabled` part in case of `SSO_PROVIDER_FROM_UMS`, similar to https://github.com/hortonworks/cloudbreak/pull/5468/files#diff-82ee44465071158ae1e7a10210eb342c.

Reply:
Will add this too.

Reply:
@dvguruprasad do you kow if we need that?

Reply:
I guess we can merge it and add it later if it turns out that we need it.

Reply:
Agreed.